src/theme/components/searchBar.ts

```diff
@@ -1,22 +1,38 @@
+import breakpoints from "../foundations/breakpoints";
 import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
 
 const { defineMultiStyleConfig, definePartsStyle } =
-  createMultiStyleConfigHelpers(["select"]);
+  createMultiStyleConfigHelpers(["button", "select"]);
 
 const SearchBar = defineMultiStyleConfig({
   baseStyle: definePartsStyle({
     display: "flex",
-    marginBottom: {
-      base: "xs",
-      md: "auto",
+    "@container (min-width: 0px)": {
+      "&": { marginBottom: "xs", flexFlow: "column nowrap" },
+      " > select": { marginBottom: "-1px" },
+      "[data-button]": {
+        borderRightRadius: "none",
+        maxWidth: "unset",
+      },
+    },
+    [`@container (min-width: ${breakpoints.md})`]: {
```

Comment:
This view is also off but it just needs a minor update for the select's right border:
<img width="592" alt="Screenshot 2025-01-29 at 9 20 11 PM" src="https://github.com/user-attachments/assets/7aabffea-9275-4289-952c-3bc3efad7156" />